Form/Extension/LayoutFormTypeExtension.php

```diff
@@ -65,9 +67,9 @@ public function buildView(FormView $view, FormInterface $form, array $options)
 
     public function finishView(FormView $view, FormInterface $form, array $options)
     {
-        if (!$view->parent && $options['compound'] && $view->vars['horizontal']) {
+        if (!$view->parent && $options['compound'] && $view->vars['layout']) {
             $class = isset($view->vars['attr']['class']) ? $view->vars['attr']['class'].' ' : '';
-            $view->vars['attr']['class'] = $class.'form-horizontal';
+            $view->vars['attr']['class'] = $class.'form-'.$view->vars['layout'];
```

Comment:
if `$view->vars['layout'] === false` this doesn't work properly? The layout variable should be one of false/null, inline, or horizontal. It should only add a class to the form if it is `inline` or `horizontal`, otherwise this will just add a class of "form-" when false or null.